src/DOM.ts

```diff
@@ -298,4 +298,33 @@ export class DOM {
     }
     return set;
   }
+
+
+  /**
+   * Heartland.DOM.addCertAlert
+   *
+   * Adds an alert letting the developer know they're in sandbox mode
+   *
+   * @param {Heartland.HPS} hps
+   * @param {string} elementid
+   */
+  public static addCertAlert() {
+
+    const el = document.createElement('div');
+    const text = document.createTextNode("You're in test mode.");
```

Comment:
Let's expand this message to explain why being in test mode is important for the consumer. Something like the below might work:

> This page is currently in test mode. Do not use real/active card numbers.

@markhagan Thoughts?